### PR TITLE
Retire OSG 3.4 (SOFTWARE-4383)

### PR DESCRIPTION
**Pull request by brianhlin**
EL6 will require more surgery so I'll toss that in a separate PR

### CODE REVIEW

File: docs/worker-node/install-wn-tarball.md
Comment (matyasselmeci) [inline docs/worker-node/install-wn-tarball.md:53]:
```suggestion
user@host $ wget https://repo.opensciencegrid.org/tarball-install/3.5/osg-wn-client-latest.el7.x86_64.tar.gz
```
while we're at it

**Review by matyasselmeci — APPROVED** [review]
Minor fix (plus the mkdocs errors if you can fix those) but otherwise looks good so pre-approving.

**Comment by brianhlin** [comment]
The CI is mad about timeouts to https://portal.xsede.org/ to https://portal.xsede.org/mfa. They look fine to me at first glance so I'm not sure if it's something that needs to be fixed in the linkchecker atm. Let's ignore them for now and treat them as transient.

I'll wait on @timtheisen's review before merging

**Review by timtheisen — APPROVED** [review]
This looks good to me.

**Comment by brianhlin** [comment]
Overriding unrelated CI failures due to XSEDE links

**Pull request merged**
